ebu_tt_live/documents/test/test_ebutt3document.py

```diff
@@ -0,0 +1,28 @@
+from unittest import TestCase
+from ebu_tt_live.documents import EBUTT3Document
+
+
+class TestEBUTT3Document(TestCase):
+
+    def test_comparison_same_sequence(self):
+        document1 = EBUTT3Document("clock", 1, "testSeq", "en-GB", "local")
+        document2 = EBUTT3Document("clock", 2, "testSeq", "en-GB", "local")
+        self.assertTrue(document1 < document2)
+        self.assertTrue(document2 > document1)
+        self.assertTrue(document1 == document1)
+        self.assertTrue(document1 <= document2)
+        self.assertTrue(document2 >= document1)
+        self.assertTrue(document1 <= document1)
+        self.assertTrue(document1 >= document1)
+        self.assertTrue(document1 != document2)
```

Comment:
It's possible to construct two document objects that have the same sequence id and number, so I think it is relevant to test - my point was just that we need to test it in both directions.